app/test_main.py

```diff
@@ -1,3 +1,48 @@
 from app.main import get_human_age
 
-# write your code here
+import pytest
+
+
+@pytest.mark.parametrize(
+    "cat_age,"
+    "dog_age,"
+    "human_age", [
+        pytest.param(
+            14, 14,
+            [0, 0],
+            id="test_14_years_cat_dog_should_convert_in_0_human_age"
+        ),
+        pytest.param(
+            15, 15,
+            [1, 1],
+            id="test_15_years_cat_dog_should_convert_in_1_human_age"
+        ),
+        pytest.param(
+            23, 23,
+            [1, 1],
+            id="test_23_years_cat_dog_should_convert_in_1_human_age"
+        ),
+        pytest.param(
+            24, 24,
+            [2, 2],
+            id="test_24_24_years_cat_dog_should_convert_in_2_human_age"
+        ),
+        pytest.param(
+            27, 28,
+            [2, 2],
+            id="test_27_28_years_cat_dog_should_convert_in_2_human_age"
+        ),
+        pytest.param(
+            28, 29,
+            [3, 3],
+            id="test_28_29_years_cat_dog_should_convert_in_3_human_age"
+        )
+    ]
```

Comment:
You've only tested the function for a very specific range of ages for cats and dogs, without checking what happens when the function receives data out of the normal range, such as negative numbers, zero, or really large numbers. These are important edge cases to consider. For example, you could add tests to check the function's behavior when it receives a negative age, zero age, or a very large age for a cat or a dog.